weis/glue_code/runWEIS.py

```diff
@@ -115,6 +115,9 @@ def run_weis(fname_wt_input, fname_modeling_options, fname_opt_options, overridd
             wt_opt = myopt.set_design_variables(wt_opt, wt_init)
             wt_opt = myopt.set_constraints(wt_opt)
             wt_opt = myopt.set_recorders(wt_opt)
+
+            if opt_options['driver']['design_of_experiments']:
```

Comment:
@nikhar-abbas shouldn't this be 

if opt_options['driver']['design_of_experiments']['flag']:

Reply:
Yes, fixed!